notebooks/MAST/tess_cutout_rotation_curve/tess_cutout_rotation_curve.ipynb

```diff
@@ -0,0 +1,810 @@
+{
```

Comment:
<p>First sentence, small adjustment: "Though <code>lightkurve</code> has a <code>search_tesscut</code> function, at the time of this tutorial (Fall 2022) it does not work for moving targets." - adding the approximate date in the text may make it easier for users to figure out if/when this functionality may change. Is there a way to put the "Last Updated" date (at the bottom of the notebook) here as well? Would just prevent users needing to scroll through the whole page to figure out what "at the time of this tutorial" means.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>I went ahead and put "September 2022" as the time of the tutorial - a little more specific than fall, and that's as specific as the Last Updated seems to be.</p>